src/Components/InputTodo.js

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { runToast } from './toast/Toast';
+
+const InputTodo = ({ list, setList }) => {
+    const [form, setForm] = useState('');
```

Comment:
`state`는 용도가 명확하게 파악되도록 이름을 짓는것이 좋습니다.
```suggestion
    const [todoInputValue, setTodoInputValue] = useState('');
```